plugins/woocommerce/src/Admin/API/OnboardingFreeExtensions.php

```diff
@@ -68,12 +73,29 @@ public function get_items_permissions_check( $request ) {
 	/**
 	 * Return available payment methods.
 	 *
-	 * @param \WP_REST_Request $request Request data.
+	 * @param WP_REST_Request $request Request data.
 	 *
-	 * @return \WP_Error|\WP_REST_Response
+	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_available_extensions( $request ) {
-		return RemoteFreeExtensions::get_extensions();
+		$extensions = RemoteFreeExtensions::get_extensions();
+		/**
+		* Remove Jetpack, WooCommerce Shipping & Tax when woocommerce_suggest_jetpack is false.
```

Comment:
Does this also remove `WooCommerce Shipping & Tax`?

Reply:
It does not remove WooCommerce Shipping & Tax. I tested and confirm both WooCommerce Shipping & Tax can be installed independently.

Reply:
Thanks for confirming! Should we just update the comment?

```suggestion
		* Remove Jetpack when woocommerce_suggest_jetpack is false.
```